src/Widgets/DisplayWidget.vala

```diff
@@ -33,10 +34,22 @@ public class Network.Widgets.DisplayWidget : Gtk.Box {
         image.icon_name = "network-wired-symbolic";
         image.icon_size = Gtk.IconSize.LARGE_TOOLBAR;
 
+        extra_info_label = new Gtk.Label (null);
+        extra_info_label.margin_left = 4;
+        extra_info_label.margin_bottom = 4;
```

Comment:
We probably shouldn't hardcode margins and instead rely on Gtk.Align.CENTER to do the right thing. Make sure you test this with the large fonts a11y setting